WordPress/Classes/Utility/ReachabilityUtils+OnlineActions.swift

```diff
@@ -35,4 +38,26 @@ extension ReachabilityUtils {
                 return notification.userInfo?[Foundation.Notification.reachabilityKey] as? Bool == true
         })
     }
+
+    /// Shows a generic non-blocking "No Connection" error message to the user.
+    ///
+    /// We use a Snackbar instead of a literal Alert because, for internet connection errors,
+    /// Alerts can be disruptive.
+    @objc static func showNoInternetConnectionNotice() {
```

Comment:
Does this need the `@objc`?

Reply:
Not sure. I opted not to analyze this. I'll check it out.